TestProjects/Playground/Playground.Uwp/project.json

```diff
@@ -1,9 +1,10 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2"
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
+    "NETStandard.Library": "2.0.1"
   },
   "frameworks": {
-    "uap10.0.10586": {}
+    "uap10.0.10240": {}
```

Comment:
Please consolidate all UWP projects to "uap10.0.10586"

Reply:
@martijn00 is there a reason we want 10586, rather than 10240?

Reply:
After this we want to update to fall creators update to support .NET Standard 2.0. Other then that we have all Uwp projects target uap10.0.10586 but set minimum to uap10.0.10240

Reply:
@martijn00 did you mean for target to be 10586 or the creators update.
Will set min to 10240